components/Footer.vue

```diff
@@ -1,34 +1,38 @@
 <template>
     <div class="footer flex border-t-4 border-secondary-bg/40 bg-secondary-bg text-sm
-    text-primary-text font-semibold py-2">
+        text-primary-text font-semibold py-2">
```

Comment:
Seems like an unrelated change?

Reply:
This probably was a lint format fix.

Reply:
As a best practice, we should be separating linting fixes from logical changes. The files have been linted before and auto-checked as part of CI/CD. Unrelated changes distract from the core purpose of this PR: cypress testing

If you confirm that there is an error with the linter (can test by running against `main`), feel free to open a separate PR to address the issue.

Reply:
Thank you for that. Previously when I was working on the other PR for the results store, I was always running into the linting issue, and was advised to run yarn lint fix, so I assumed that what I should always do before a push. I'll keep that in mind for the future